core/iwasm/aot/arch/aot_reloc_thumb.c

```diff
@@ -97,8 +97,24 @@ static SymbolMap target_sym_map[] = {
     /* clang-format off */
     REG_COMMON_SYMBOLS
     /* compiler-rt symbols that come from compiler(e.g. gcc) */
+#if __ARM_ARCH != 6
```

Comment:
Do these sysbols really not needed by arm 6? Will it cause relocation errors in some boards?

Reply:
It seems that armv6m's libgcc.a don't provide these symbols, when I try to build it, I got a error "undefined reference to [these symbol]".